earthpy/tests/test_plot_bands.py

```diff
@@ -0,0 +1,217 @@
+    """Test the number of axes.
+
+    If provided with a 2 band array, plot_bands should return 3 axes.
+    And 2 colorbars
+    """
+    ax = ep.plot_bands(image_array_2bands)
+    ax = list(ax)
+    cb = [a.images[0].colorbar for a in ax if a.images]
+
+    assert len(ax) == 3
+    assert len(cb) == 2
+    plt.close()
+
+
+def test_two_plot_title(image_array_2bands):
+    """Test that the default title is provided for a 2 band array plot."""
+
+    ax = ep.plot_bands(image_array_2bands)
+    num_plts = image_array_2bands.shape[0]
+    all_titles = [ax[i].get_title() for i in range(num_plts)]
+    assert all_titles == ["Band 1", "Band 2"]
+    plt.close()
+
+
+def test_custom_plot_title(image_array_2bands):
+    """Test that the custom title is applied for a 2 band array plot."""
+    ax = ep.plot_bands(image_array_2bands, title=["Red Band", "Green Band"])
+    num_plts = image_array_2bands.shape[0]
+    all_titles = [ax[i].get_title() for i in range(num_plts)]
+    assert all_titles == ["Red Band", "Green Band"]
+    plt.close()
+
+
+def test_single_band_3dims(one_band_3dims):
+    """Test single band plot with three dimensions.
+
+    If you provide a single band array with 3 dimensions (shape[0]==1
+    test that it still plots and only returns a single axis.
+    """
+    ax = ep.plot_bands(one_band_3dims)
+    arr = ax.get_images()[0].get_array()
+    assert arr.ndim == 2
+    assert len(ax.get_images()) == 1
+    plt.close()
+
+
+def test_single_band_2dims(one_band_3dims):
+    """Test single band plot with two dimensions
+
+    If you provide a single band array with 3 dimensions (shape[0] == 1)
+    test that it still plots and only returns a single axis.
+    """
+    single_band_2dims = one_band_3dims[0]
+    ax = ep.plot_bands(single_band_2dims)
+    # Get array from mpl figure
+    arr = ax.get_images()[0].get_array()
+    assert arr.ndim == 2
+    assert len(ax.get_images()) == 1
+    plt.close()
+
+
+def test_cbar_param(one_band_3dims):
+    """Test that the colorbar param works for a single band arr
+    """
+    one_band_2dims = one_band_3dims[0]
+    ax = ep.plot_bands(one_band_2dims)
+    arr = ax.get_images()[0].get_array()
+    c_bar = ax.images[0].colorbar
+
+    # Return arr should be scaled by default between 0-255
+    assert arr.min() == 0 and arr.max() == 255
+    # A cbar should be drawn in this plot
+    assert c_bar
+    plt.close()
+
+
+def test_not_scaled_single_band(one_band_3dims):
+    """Test if the user turns off scaling and cbar the data vals should remain intact.
+
+    Also if no cbar is specified it should not render.
+    """
+    one_band_2dims = one_band_3dims[0]
+
+    ax = ep.plot_bands(one_band_2dims, cbar=False, scale=False)
+    arr = ax.get_images()[0].get_array()
+    c_bar = ax.images[0].colorbar
+
+    # Return arr is unscaled for plotting
+    assert (
+        arr.min() == one_band_2dims.min() and arr.max() == one_band_2dims.max()
+    )
+    # A cbar should be drawn in this plot
```

Comment:
Maybe a copy paste error here?

```suggestion
    # A cbar should not be drawn in this plot
```